tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1392,6 +1392,26 @@ void test_localized_char() {
     assert(format(STR("{:Lc}"), T('c')) == STR("c"));
 }
 
+template <class T>
+struct Box {
+    T value;
+};
+
+template <class T, class charT>
+struct std::formatter<Box<T>, charT> : std::formatter<T, charT> {
+    template <class FormatContext>
+    auto format(Box<T> t, FormatContext& fc) const {
+        return formatter<T, charT>::format(t.value, fc);
+    }
+};
+
+// Also test GH-2765 "<format>: Cannot format a long value with formatter"
+template <class charT>
+void test_gh_2765() {

```

Comment:
I think this test is better suited for [`P0645R10_text_formatting_custom_formatting`](https://github.com/microsoft/STL/blob/main/tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp). It can also reuse the mechanism defined there.

Reply:
Ah, excellent catch!